TravelPro.html

```diff
@@ -0,0 +1,295 @@
+<!DOCTYPE html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TravelPro</title>
+    <link rel="stylesheet" href="TravelPro.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+    <div class="navbar">
+        <div id="logo">
+            <img src="./images/logo.png" alt="logo" height="80px" width="110px">
+        </div>
+        <div id="options">
+            <a>Destinations</a>
```

Comment:
The navigation links `<a>Destinations</a>` etc., lack href attributes. If they are placeholders, consider using href="#" for the time being.